pool/queue/repo.ml

```diff
@@ -160,18 +160,24 @@ let find label id =
   ||> CCOption.to_result Pool_common.Message.(NotFound Field.Queue)
 ;;
 
+let workable_query =
```

Comment:
That could directly be integrated in `find_workable_request`

Reply:
this one is used by the count at the file end as well. I combined the query generation.